pyfarm/master/entrypoints.py

```diff
@@ -1,6 +1,7 @@
 # No shebang line, this module is meant to be imported
 #
 # Copyright 2013 Oliver Palmer
+# Copyright 2014, 2015 Ambient Entertainment GmbH & Co. KG
```

Comment:
Hmm, first contribution here was in 2014 unless I'm mistaken.  Since we don't add copyrights for every year a file is contributed to, why the entry for 2015?